js/window.ts

```diff
@@ -1,10 +1,6 @@
 // Copyright 2018-2019 the Deno authors. All rights reserved. MIT license.
 
-// (0, eval) is indirect eval.
 // See the links below for details:
 // - https://stackoverflow.com/a/14120023
```

Comment:
These two links no longer make sense given the current solution, they talk about indirect eval.

Reply:
Can be safely removed, right?

Reply:
It doesn’t seem to work with snapshots..

Reply:
Yeah, that was the problem from before.